src/components/Button/index.tsx

```diff
@@ -1,19 +1,26 @@
 /** @jsx jsx */
 import * as React from "react";
-import { jsx, css } from "@emotion/core";
-import t from "../../../lib/theme";
+import { jsx } from "@emotion/core";
+import { Theme } from "../../types/global";
+import Text from "../Text";
```

Comment:
We should consider absolute paths at some point before a refactor becomes a large undertaking.